src/components/SearchForm.vue

```diff
@@ -0,0 +1,180 @@
+<template>
+  <div class="autocomplete">
+    <input class="form-control" type="text"
+           v-model="queryText"
+           @input="updateQueryText($event.target.value)"
+           v-on:input="refreshSuggestions"
+           @keydown.enter='enter'
+           @keydown.down='down'
+           @keydown.up='up'
+           placeholder="Search..."
+    >
+    <div class="dropdown-pane" :class="{'is-open': hasSuggestions }" data-dropdown>
+      <ul>
+        <li v-for="(suggestion, index) in suggestions"
+            :key="suggestion.id"
+            :class="{'highlighted': isActive(index)}"
+            @click="suggestionClick(index)"
+            @mouseover="mouseOver(index)"
+            @mouseout="mouseOut()"
+        >
+          <strong>{{ suggestion.exact_synonym[0] }}</strong><br/>
```

Comment:
Is the space after `{{` and before `}}` also a default style?

Reply:
This is style I copied from the component this is based on ([see here](https://github.com/putmantime/vue_autocomplete/blob/master/src/components/AutoComplete.vue)), but I don't feel strongly about it. ESLint isn't complaining if I remove the spaces, so I'm happy to do that if it's more readable.

Reply:
Oh, and it's used in the Vue documentation as well: https://vuejs.org/v2/guide/#Declarative-Rendering